### DISCUSSION TITLE
shigan/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#102] [Giscus]
# shigan/

写得很好，很细腻，也解惑。分享留存。 Q：昨天我们这里一个十三岁的女孩跳楼了，为什么现在的年轻人心理承受能力如此差？ A：知乎 Miss.奇迹 关于

https://www.eallion.com/shigan/

**Comment by botsp** [comment]
写的真好。
网络居民很容易接触到各种各样的智慧哲理与人生规律，但什么真正刻画着我们的存在? 恐怕不是脑子里装了多少人生攻略，而是自我体验与周围世界的联系，这才是真正的"我"。
自己逐渐想通了这些问题，可是回头一看，新生代小孩子问题似乎更严重了。

**Reply by eallion** [reply]
现在会更多时间带小孩出去与真实世界接触了。
